pkg/api/constants/constants.go

```diff
@@ -0,0 +1,36 @@
+package constants
+
+const (
+	LivenessEndpoint           = "/_health/liveness"
+	VersionEndpoint            = "/api/version"
+	ComponentsEndpoint         = "/components"
+	InbandInventoryEndpoint    = "/inventory/in-band"
+	OutofbandInventoryEndpoint = "/inventory/out-of-band"
+
+	// server service attribute to look up the BMC IP Address in
+	BmcAttributeNamespace = "sh.hollow.bmc_info"
+
+	// server server service BMC address attribute key found under the bmcAttributeNamespace
+	BmcIPAddressAttributeKey = "address"
+
+	// serverservice namespace prefix the data is stored in.
+	ServerServiceNSPrefix = "sh.hollow.alloy"
```

Comment:
use constants from rivets

Reply:
Good catch and seems like we do not need this file anymore. I just deleted.